### PR TITLE
Admin - Ability to enable/disable user

### PR DESCRIPTION
**Pull request by EvyW**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -26,3 +26,7 @@
 // import custom scss, content to be set in deployment.
 //
 @import "tail";
+
+.modal-footer form{
+  margin: 0;
+}
```
Comment (elijh) [inline app/assets/stylesheets/application.scss]: The comment in this file should make it more clear, but nothing should come after `@import "tail"`. The purpose of importing tail.scss is to all someone deploying the app to override all other css, but they can't do this if there is css after the import.

the proper place to add css is in leap.scss

**Comment by elijh** [comment]
Looks great. Maybe also add enabled/disable buttons to `views/users/_user.html.haml`? (but only visible if the user is an admin).

File: config/locales/es.yml [hunk 2]
```diff
@@ -110,6 +110,14 @@ es:
...
         one: Usuario
         other: '%{count} usuarios'
+  actions:
+    toggle_user: "Enable / Disable User"
+    enable_user: "Enable User"
+    disable_user: "Disable User"
+    confirm_user_deactivation: "Estás seguro que quieres desactivar user con username '%{username}'?"
+    confirm_user_activation: "Estás seguro que quieres activar user con username '%{username}'?"
+    user_disabled_message: "User con username '%{username}' ha sido desactivado."
+    user_enabled_message: "User with username '%{username}' ha sido activado."
```
Comment (azul) [inline config/locales/es.yml]: :mag: minor: 'with' is translated in the disable message but not the enable message.

**Comment by azul** [comment]
hey @EvyW, loooks great! Could you revert the change to the application.scss file?
The corresponding lines now are in leap.scss but they should also be cleared from there.
Then i think this is ready to go :shipit: :+1:

File: config/locales/es.yml (hunk 2 above)
Comment (EvyW) [inline config/locales/es.yml]: Done, but there were missing more translations, so, I have translated some words from line 114 to line 120 https://github.com/EvyW/leap_web/blob/develop/config/locales/es.yml

**Comment by EvyW** [comment]
Sure, it's ready https://github.com/EvyW/leap_web/blob/develop/app/assets/stylesheets/application.scss

File: app/views/users/_user.html.haml [hunk 3]
```diff
@@ -1,4 +1,32 @@
-%tr
-  %td= link_to user.login, user
-  %td= l(user.created_at, :format => :short)
-  %td= l(user.updated_at, :format => :short)
+      %tr
```
Comment (azul) [inline app/views/users/_user.html.haml]: haml does not like the indenting at the start of the document. That's also why some of the tests are failing now.

Please make sure to start with no indenting:
```haml
%tr
  %td= link_to user.login, user
  %td= ...
```

**Comment by azul** [comment]
@EvyW getting close :smile: . There are some haml issues in the _users haml file still. They are also breaking the tests :red_circle:

**Comment by EvyW** [comment]
Hi @azul :) Does it look better?? https://github.com/leapcode/leap_web/commit/289424bda4cabfa682e07059bfdc1a0c7f04c1ef

**Comment by azul** [comment]
oh... sorry. my comment was confusing. The problem is the indenting of the first line. It should start at the first char of the line. Otherwise haml will complain. I don't care about the td and = identing too much. Should have left that the same.

**Comment by EvyW** [comment]
aaah ok, @azul I think now is better https://github.com/EvyW/leap_web/commit/df62d292f5df6dffd8a0e621181632f276c0b304 
td and = are ok?? BTW thank you for your help  :smiley:

**Comment by elijh** [comment]
actually, the new change does not fix the problem, and introduces new problems :(

the problem is that the first tr is indented. this is not allowed.

````
    tr%
````

must be this:

````
%tr
````

But, once you remove the indent for %tr, then everything below it must also be "outdented" exactly the same amount. It is illegal to nest an element more than two spaces.

For more information on HAML, there are nice guides online, maybe start here http://haml.info/tutorial.html

But also, if you try to run the app with these view templates, the rails error will tell you want is wrong.

**Comment by azul** [comment]
Hey @EvyW ,
thanks a lot. This looks good. :+1:

**Comment by EvyW** [comment]
You're welcome! Let me know if is need it something else  :smiley:

**Pull request merged**